data-serving/scripts/setup-db/schemas/sources.schema.json

```diff
@@ -28,10 +28,10 @@
                 "bsonType": "string"
             },
             "excludeFromLineList": {
-                "bsonType": "boolean"
+                "bsonType": "bool"
```

Comment:
Is this due to migrate-mongo?

Reply:
I'm not sure why this worked before if it did work before: boolean isn't a valid bson type alias.